src/components/billing/payment-modal/index.tsx

```
@@ -26,19 +28,38 @@ export default function PaymentModal({
-  const finalAmount = Math.max(0, (plan?.price || 0) - pointsToUse); // 최종 결제 금액
+  
+  const promotionDiscount = appliedPromotion?.discount || 0;
+  const finalAmount = Math.max(0, (plan?.price || 0) - pointsToUse - promotionDiscount); // 최종 결제 금액
 
   const handleUseAllPoints = () => {
     setPointsToUse(Math.min(userPoints, plan?.price || 0));
   };
 
+  const handleApplyPromotion = () => {
+    const validPromotion = validatePromotionCode(promotionCode);
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)